### PR TITLE
Solved #71 Issue

### LINKED ISSUE #71: Remove the gap
**Describe the bug**
There is Empty space between the "about" section and the "contributors" section

**Expected behavior**
Remove the Empty space between the "about" section and the "contributors" section.

**Screenshots**
Screenshot 2023-03-16 134217

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by munnokd**
## Purpose

The purpose of this PR is to fix #71

## Approach
I made a single change in CSS file. There was padding bottom so I add padding bottom 0px.
## Screenshot
image

### CODE REVIEW

**Comment by Tharindu37** [comment]
@munnokd Screenshot a required. Please edit the PR

**Comment by munnokd** [comment]
okay

**Comment by munnokd** [comment]
@Tharindu37 now check the screenshot I removed the unnecessary gap.

**Comment by Tharindu37** [comment]
@munnokd Can you remove `  padding: 5%;` this line

File: styles.css [hunk 1]
```diff
@@ -54,6 +54,7 @@ html {
 }
 .about {
   padding: 5%;
+  padding-bottom: 0px;
```
Comment (Tharindu37) [inline styles.css:57]:
```suggestion
```
Remove this lines

**Comment by munnokd** [comment]
@Tharindu37 Now check.

**Comment by Tharindu37** [comment]
@munnokd Thank you

**Pull request merged**